### PR TITLE
Module content

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

File: upload-plugin/src/main/groovy/ly/count/android/plugins/uploadSymbols.groovy [hunk 1]
```diff
@@ -58,27 +55,17 @@ class UploadSymbolsPlugin implements Plugin<Project> {
             .build()
         request = new Request.Builder().url(url).post(formBody).build()
       }
+      logger.debug("uploadJavaSymbols, Generated request: {}", request.body().toString())
```
Comment (turtledreams) [inline upload-plugin/src/main/groovy/ly/count/android/plugins/uploadSymbols.groovy:58]: u sure?

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: upload-plugin/src/main/groovy/ly/count/android/plugins/uploadSymbols.groovy (hunk 1 above)
Comment (arifBurakDemiray) [reply upload-plugin/src/main/groovy/ly/count/android/plugins/uploadSymbols.groovy:58]: ups, why this is here and seems like a new thing

Comment (arifBurakDemiray) [reply upload-plugin/src/main/groovy/ly/count/android/plugins/uploadSymbols.groovy:58]: sumimasee, this pr become a godzilla

**Comment by arifBurakDemiray** [comment]
The failing test is successful when run alone or when run with tha same tests in the file. But when all tests run it fails for github actions. However, it does not fail on my end

**Review by turtledreams — CHANGES_REQUESTED** [review]
when logging a variable pls envelop it with brackets to we can see the entirety of it in weird cases

**Pull request merged**